src/app/(proper_react)/(redesign)/(public)/RebrandAnnouncement.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./RebrandAnnouncement.module.scss";
+import { CloseBtn } from "../../../components/server/Icons";
+import { useL10n } from "../../../hooks/l10n";
+import { useLocalDismissal } from "../../../hooks/useLocalDismissal";
+import { useHasRendedClientSide } from "../../../hooks/useHasRenderedClientSide";
+
+export const RebrandAnnouncement = () => {
+  const l10n = useL10n();
+  const dismissal = useLocalDismissal("mozilla-monitor-rebrand", {
+    duration: Number.MAX_SAFE_INTEGER,
```

Comment:
What is the `duration` for here? Auto-closing?
Acccctually, per a doc comment below:
> ```js
> /** How long the dismissal should last, in seconds. Note that by default, the cookie will be deleted in 30 days. */
> ```

Where [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) says the value of `Number.MAX_SAFE_INTEGER` is ~9 quadrillion; and Google says 
that [9,007,199,254,740,991 seconds to years](https://www.google.com/search?q=9%2C007%2C199%2C254%2C740%2C991+s+to+years) is approximately 285_616_414 years (give or take a couple months).

Reply:
Yep correct. I might've gone overboard with `MAX_SAFE_INTEGER` here, not showing it again for a 100 years would probably have been fine too :sweat_smile: